From 816532f7bcd2dfcacc88438b8438b15a03a4811e Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 14 Nov 2023 18:22:58 +0100
Subject: [PATCH 0036/1695] core: arm: fix inline comment on async notif
 interrupt

Fixes the inline comment that describes allowed values for
CFG_CORE_ASYNC_NOTIF_GIC_INTID that can be a SPI or a secure PPI.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 228efa3dbba3bfd8e0af28008952c8eca45d393c core: arm: fix inline comment on async notif interrupt]
Change-Id: Ia149c5303b8e48c0f2b1347755b3b6b8e5519d96
Fixes: 943972855082 ("core: notif: allow GIC_PPI usage for async notif")
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 3151cd70de739da3e17db98bf454086765683fb0]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/arm.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/arm.mk b/core/arch/arm/arm.mk
index 706556b94..7ff1c1ae6 100644
--- a/core/arch/arm/arm.mk
+++ b/core/arch/arm/arm.mk
@@ -179,7 +179,8 @@ CFG_SM_NO_CYCLE_COUNTING ?= y
 # CFG_CORE_ASYNC_NOTIF_GIC_INTID is defined by the platform to some free
 # interrupt. Setting it to a non-zero number enables support for using an
 # Arm-GIC to notify normal world. This config variable should use a value
-# larger the 32 to make it of the type SPI.
+# larger or equal to 24 to make it of the type SPI or PPI (secure PPI
+# only).
 # Note that asynchronous notifactions must be enabled with
 # CFG_CORE_ASYNC_NOTIF=y for this variable to be used.
 CFG_CORE_ASYNC_NOTIF_GIC_INTID ?= 0
-- 
2.47.2

